flink-scala-shell/src/main/scala/org/apache/flink/api/scala/FlinkShell.scala

```diff
@@ -277,8 +277,9 @@ object FlinkShell {
 
     val cluster = clusterDescriptor.deploySessionCluster(clusterSpecification)
 
-    val inetSocketAddress = AkkaUtils.getInetSocketAddressFromAkkaURL(
-      cluster.getClusterConnectionInfo.getAddress)
```

Comment:
Did this ever work with Flip-6?

Reply:
I don't know whether there is any user connect YARN cluster by Scala Shell. Theoretically in standalone ha(a.k.a. non ha) it is no difference with WebMonitor. But with ZooKeeper enabled it will retrieve `Dispatcher` address and later it should failed to send REST request to `Dispatcher`.